src/lib/date.ts

```diff
@@ -261,11 +261,17 @@ export function cascadingFormattedStartDateTime(
   return dateRange(startAt, endAt, timezone, "long")
 }
 
-export function auctionsDetailFormattedStartDateTime(startAt, endAt, endedAt) {
+export function auctionsDetailFormattedStartDateTime(
+  startAt,
+  endAt,
+  endedAt,
+  liveStartAt
+) {
   const thisMoment = moment.tz(moment(), "GMT")
   const saleStartMoment = moment.tz(startAt, "GMT")
   const lotsClosingMoment = moment.tz(endAt, "GMT")
   const saleEndMoment = moment.tz(endedAt, "GMT")
+  const liveStartMoment = moment.tz(liveStartAt, "GMT")
 
   if (!!endedAt)
```

Comment:
do you need the double `!`?

Reply:
yeah that forces it into a boolean I believe.